### PR TITLE
Feature/intygfv 14393

### PR DESCRIPTION
**Pull request by IamFriberg**

### CODE REVIEW

File: persistence/src/main/resources/changelog/db-changelog-master.xml [hunk 1]
```diff
@@ -143,4 +143,37 @@
 
   </changeSet>
 
+  <changeSet id="8" author="Mikael Heglind">
```
Comment (mww-sopra-steria) [inline persistence/src/main/resources/changelog/db-changelog-master.xml:146]: Can be removed

File: test/build.gradle.kts [hunk 2]
```diff
@@ -8,7 +8,7 @@ plugins {
 node {
   version = Dependencies.nodeVersion
   download = true
-  distBaseUrl = "https://nodejs.org/dist/"
+  distBaseUrl = "https://build-inera.nordicmedtest.se/node/"
```
Comment (mww-sopra-steria) [inline test/build.gradle.kts:11]: Should be nodejs.org/dist

File: web/client/src/components/dataExport/DataExportStatus.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const DataExportStatus = {
```
Comment (mww-sopra-steria) [inline web/client/src/components/dataExport/DataExportStatus.js:1]: This can be removed

File: web/client/src/store/configureStore.js [hunk 4]
```diff
@@ -10,7 +10,8 @@ export default function configureStore(preloadedState) {
   const middlewares = [thunk, routerMiddleware(history)];
   const middlewareEnhancer = applyMiddleware(...middlewares);
 
-  const enhancers = [middlewareEnhancer];
+  const enhancers = [middlewareEnhancer,
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()]; //TODO: Added to make Redux DevTools work
```
Comment (mww-sopra-steria) [inline web/client/src/store/configureStore.js:14]: Should be removed.

**Review by mww-sopra-steria — APPROVED** [review]
Nice! Few comments to remove some code.

File: persistence/src/main/resources/changelog/db-changelog-master.xml (hunk 1 above)
Comment (IamFriberg) [reply persistence/src/main/resources/changelog/db-changelog-master.xml:146]: Fixed!

File: test/build.gradle.kts (hunk 2 above)
Comment (IamFriberg) [reply test/build.gradle.kts:11]: Fixed!

File: web/client/src/components/dataExport/DataExportStatus.js (hunk 3 above)
Comment (IamFriberg) [reply web/client/src/components/dataExport/DataExportStatus.js:1]: Fixed!

File: web/client/src/store/configureStore.js (hunk 4 above)
Comment (IamFriberg) [reply web/client/src/store/configureStore.js:14]: Fixed!

**Pull request merged**